pkg/manager/manager.go

```diff
@@ -314,6 +314,19 @@ type LeaderElectionRunnable interface {
 	NeedLeaderElection() bool
 }
 
+// WarmupRunnable knows if a Runnable should be a warmup runnable. A warmup runnable is a runnable
+// that should be run when the manager is started, but before the leader election is acquired.
+// Note: Implementing this interface is only useful when LeaderElection is enabled, as the behavior
+// without leaderelection is to run LeaderElectionRunnables immediately.
+type WarmupRunnable interface {
+	// Warmup is the implementation of the warmup runnable.
```

Comment:
The comment doesn't really say much :)

Reply:
Changed, let me know if there's anything else you want me to add